### ISSUE TITLE
Division by float 0 in rasterizer_canvas_batcher.h(image height or width are equal to 0)

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#43772]
**Godot version:**
3.2.4.beta.custom_build. b900ec03f

**OS/device including version:**
Ubuntu 20.04

**Issue description:**
When I run project, then I see this undefined sanitizers errors
```
drivers/gles_common/rasterizer_canvas_batcher.h:909:40: runtime error: division by zero
drivers/gles_common/rasterizer_canvas_batcher.h:910:40: runtime error: division by zero
```
https://github.com/godotengine/godot/blob/b900ec03f0d653f76b435bd9ce3a62ef38cffc78/drivers/gles_common/rasterizer_canvas_batcher.h#L909-L910

**Steps to reproduce:**
1. Download project
2. Go to project folder
3. Download and run program - https://github.com/qarmin/godot_image_replacer/releases/download/0.1.0/godot_image_replacer - inside it (this will replace all images with new one with random color and 1x1 dimension to speedup project importing)
4. Run project with undefined sanitizers support

**Minimal reproduction project:**
https://github.com/Orama-Interactive/Pixelorama

**Comment by lawnjelly** [comment]
I'll put a check in, this has come up before (or similar) as the legacy renderer is full of these (1/width etc). At some points we don't do the division by zero check deliberately, but it should be ok here.

Just to be clear, I haven't run your project, but can textures be created with zero width / height (I don't know this area of godot)? Or would the texture creation fail?

**Comment by akien-mga** [comment]
Fixed by #43780.
